presets/4.3/tune/fpvian_basher_freestyle.txt

```diff
@@ -0,0 +1,181 @@
+#$ TITLE: Basher Freestyle 5" by FPVian
```

Comment:
i think its worth noting in the title if that's for 6S or 4S?
or is that for both?

Reply:
Added 6S to title